frontend/src/summary.pug

```diff
@@ -90,34 +90,38 @@
       .error-message-box__failed-repo--reason(v-else) {{ errorBlock.errorMessage }}
   #summary-charts
     .summary-charts(v-for="(repo, i) in filteredRepos")
-      .summary-charts__title(v-if="filterGroupSelection === 'groupByRepos'")
-        .summary-charts__title--index {{ i+1 }}
-        .summary-charts__title--repo {{ repo[0].repoName }}
-        .summary-charts__title--contribution.mini-font(
-          title="Total contribution of group"
-        ) [{{ getGroupTotalContribution(repo) }} lines]
-        template(v-if="isMergeGroup")
-          a(
-            v-bind:href="getRepoLink(repo[0])", target="_blank",
-            title="click to view group's repo"
-          )
-            i.icon-button.fab.fa-github
-        .summary-charts__title--percentile.mini-font {{ getPercentile(i) }} %
       .summary-charts__title(
-        v-if="filterGroupSelection === 'groupByAuthors'",
-        v-bind:class="{ warn: repo[0].name === '-' }"
+        v-if="filterGroupSelection === 'groupByRepos' || filterGroupSelection === 'groupByAuthors'"
```

Comment:
Note that I also refactored the structure of the `.summary-charts__title`. 

Previously, `.summary-charts__title` is separated into two, each for when filterGroupSelection is `groupByAuthors` or `groupByRepos`. There is overlapping code among the two, and separating into two makes it hard to understand the dynamic parts of the code.

If this changes is not needed or should be done in another PR, do let me know and I will act accordingly.